photon-core/src/main/java/org/photonvision/vision/pipeline/CVPipelineSettings.java

```diff
@@ -36,8 +36,8 @@
     @JsonSubTypes.Type(value = ObjectDetectionPipelineSettings.class)
 })
 public class CVPipelineSettings implements Cloneable {
-    public int pipelineIndex = 0;
-    public PipelineType pipelineType = PipelineType.DriverMode;
+    @SuppressSettingCopy public int pipelineIndex = 0;
```

Comment:
wait actually. why do we not wanna copy index? don't we?

Reply:
we end up overwriting it anyway

Reply:
Right, so why skip it and then explicitly copy it? Why not do neither?